src/tests/ftest/launch.py

```diff
@@ -1875,6 +1877,9 @@ def _prepare(self, test, repeat, user_create):
         if status:
             return status
 
+        # Stop rogue processes
+        self._cleanup_procs(test, True)
+
```

Comment:
We should probably only call this after the test to avoid essentially duplicate calls.
```suggestion
```

Reply:
Done